src/plugins/intel_gpu/src/runtime/ocl/ocl_device.cpp

```diff
@@ -33,10 +35,13 @@
 #include <dlfcn.h>
 #endif
 
+
 namespace cldnn {
 namespace ocl {
 
 namespace {
+std::mutex deviceInitCacheAccessMutex;
```

Comment:
I think this can be a static variable within the function.

Reply:
Moved.